src/main/java/io/github/jancieslak/tba/dto/PostReserveScreeningResponseModel.java

```diff
@@ -0,0 +1,29 @@
+package io.github.jancieslak.tba.dto;
+
+import java.time.LocalDateTime;
+
+public class PostReserveScreeningResponseModel {
+    private float toPay;
+    private LocalDateTime expiresAt;
+
+    public PostReserveScreeningResponseModel(float toPay, LocalDateTime expiresAt) {
```

Comment:
duży minus za float do trzymania $$

Reply:
Mój błąd, powinienem to trzymać jako int